src/components/questionTypes/FREE/creation/options/Options.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+
+import React from 'react'
+import classNames from 'classnames'
+import { FormattedMessage } from 'react-intl'
+
+import type { TextInputType } from '../../../../../types'
+
+type Props = {
+  intl: $IntlShape,
+  input: TextInputType,
```

Comment:
TextInputType is wrong for this kind of component (will result in flow errors). It expects value to be a string and onChange to update this string. You will need to write something like:

`input: {
  value: {
    restrictions: {
      type...
      min....
    }
  },
  onChange: (any) => void,
}`